smokes/master.cfg

```diff
@@ -115,5 +113,6 @@ c['buildbotNetUsageData'] = None
 c['db'] = {
     # This specifies what database buildbot uses to store its state.  You can leave
     # this at its default for all but the largest installations.
-    'db_url' : "sqlite:///state.sqlite",
+    'db_url': "sqlite:///state.sqlite",
 }
+exec(open("mydashboard.py").read())
```

Comment:
Is it original `master.cfg` ? One new line with comments before this exec would  be helpfull